.github/workflows/dependency-check.yml

```
@@ -0,0 +1,43 @@
+name: Suppressions
+
+on:
+  schedule:
+    - cron: "05 8-16/3 * * 1-5" # On the 5th minute, every 3 hours, from 8am to 4pm
```

I think we should do this in a wider time period, to reduce the risk of nightly pipelines failing as well - every 3 hours sounds reasonable though!
```suggestion
    - cron: "05 6-18/3 * * 1-5" # On the 5th minute, every 3 hours, from 8am to 4pm
```